introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb

```diff
@@ -0,0 +1,733 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "import time\n",
+    "\n",
+    "timestamp = time.strftime('-%Y-%m-%d-%H-%M-%S', time.gmtime())\n",
+    "endpoint_name = job_name_prefix + '-ep-' + timestamp\n",
+    "print('Endpoint name: {}'.format(endpoint_name))\n",
+    "\n",
+    "endpoint_params = {\n",
+    "    'EndpointName': endpoint_name,\n",
+    "    'EndpointConfigName': endpoint_config_name,\n",
+    "}\n",
+    "endpoint_response = sagemaker.create_endpoint(**endpoint_params)\n",
+    "print('EndpointArn = {}'.format(endpoint_response['EndpointArn']))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, now the endpoint can be created. It may take sometime to create the endpoint..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# get the status of the endpoint\n",
+    "response = sagemaker.describe_endpoint(EndpointName=endpoint_name)\n",
+    "status = response['EndpointStatus']\n",
+    "print('EndpointStatus = {}'.format(status))\n",
+    "\n",
+    "\n",
+    "# wait until the status has changed\n",
+    "sagemaker.get_waiter('Endpoint_Created').wait(EndpointName=endpoint_name)\n",
+    "\n",
+    "\n",
+    "# print the status of the endpoint\n",
+    "endpoint_response = sagemaker.describe_endpoint(EndpointName=endpoint_name)\n",
+    "status = endpoint_response['EndpointStatus']\n",
+    "print('Endpoint creation ended with EndpointStatus = {}'.format(status))\n",
+    "\n",
+    "if status != 'InService':\n",
+    "    raise Exception('Endpoint creation failed.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you see the message,\n",
+    "\n",
+    "> `Endpoint creation ended with EndpointStatus = InService`\n",
+    "\n",
+    "then congratulations! You now have a functioning inference endpoint. You can confirm the endpoint configuration and status by navigating to the \"Endpoints\" tab in the AWS SageMaker console.\n",
+    "\n",
+    "We will finally create a runtime object from which we can invoke the endpoint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Perform inference\n",
+    "Finally, the customer can now validate the model for use. They can obtain the endpoint from the client library using the result from previous operations, and generate classifications from the trained model using that endpoint.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "runtime = boto3.Session().client(service_name='sagemaker-runtime') "
```

Comment:
I think this has to be "runtime.sagemaker" to work outside of a SageMaker Notebook Instance.

Reply:
Changed.